Files/CueSheet.Deserializer.cs

```diff
@@ -168,7 +168,8 @@ public partial class CueSheet : IFileSerializer<Models.CueSheets.CueSheet>
                     // Default means return
                     default:
                         i--;
-                        return null;
+                        cueFile.Tracks = cueTracks.ToArray();
```

Comment:
If this and the other change are related to a next line being hit errantly, then I would prefer that a separate, commented case is used instead of overloading the default case.